Exception/AbstractAlgoliaProductException.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Module Algolia Algoliasearch
+ */
+namespace Algolia\AlgoliaSearch\Exception;
+
+use Magento\Catalog\Model\Product;
+
+/**
+ * Class: AbstractAlgoliaProductException
+ */
+abstract class AbstractAlgoliaProductException extends \Exception
+{
+    /**
+     * @var Product
+     */
```

Comment:
Could you "one-line" those annotations to /** @var Product */?
The same applies to all one line annotations / comments :)